[2022-04-24 02:21:41,067] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.crash_local_to_gcs_task manual__2022-04-24T01:52:31.743537+00:00 [queued]>
[2022-04-24 02:21:41,187] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.crash_local_to_gcs_task manual__2022-04-24T01:52:31.743537+00:00 [queued]>
[2022-04-24 02:21:41,189] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 02:21:41,192] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2022-04-24 02:21:41,193] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 02:21:41,277] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): crash_local_to_gcs_task> on 2022-04-24 01:52:31.743537+00:00
[2022-04-24 02:21:41,320] {standard_task_runner.py:52} INFO - Started process 274 to run task
[2022-04-24 02:21:41,363] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'local_2_gcs_2_bq_dag', 'crash_local_to_gcs_task', 'manual__2022-04-24T01:52:31.743537+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/download_2_gcs_dag_V2.py', '--cfg-path', '/tmp/tmpliderhok', '--error-file', '/tmp/tmpft3ll555']
[2022-04-24 02:21:41,371] {standard_task_runner.py:77} INFO - Job 62: Subtask crash_local_to_gcs_task
[2022-04-24 02:21:41,966] {logging_mixin.py:109} INFO - Running <TaskInstance: local_2_gcs_2_bq_dag.crash_local_to_gcs_task manual__2022-04-24T01:52:31.743537+00:00 [running]> on host 1f2512cca76d
[2022-04-24 02:21:42,311] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_2_gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=crash_local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-24T01:52:31.743537+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-24T01:52:31.743537+00:00
[2022-04-24 02:24:13,797] {python.py:175} INFO - Done. Returned value was: None
[2022-04-24 02:24:13,842] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=local_2_gcs_2_bq_dag, task_id=crash_local_to_gcs_task, execution_date=20220424T015231, start_date=20220424T022141, end_date=20220424T022413
[2022-04-24 02:24:13,984] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-24 02:24:14,145] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
